3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/MailchimpAdapter.scala

```diff
@@ -98,10 +98,10 @@ object MailchimpAdapter extends Adapter {
    */
   def toRawEvents(payload: CollectorPayload)(implicit resolver: Resolver): ValidatedRawEvents =
     (payload.body, payload.contentType) match {
-      case (None, _)                          => s"Request body is empty: no ${VendorName} event to process".failNel
-      case (_, None)                          => s"Request body provided but content type empty, expected ${ContentType} for ${VendorName}".failNel
-      case (_, Some(ct)) if ct != ContentType => s"Content type of ${ct} provided, expected ${ContentType} for ${VendorName}".failNel
-      case (Some(body), _)                    => {
+      case (None,       _) => s"Request body is empty: no ${VendorName} event to process".failNel
+      case (_,          None) => s"Request body provided but content type empty, expected ${ContentType} for ${VendorName}".failNel
+      case (_,          Some(ct)) if ct != ContentType => s"Content type of ${ct} provided, expected ${ContentType} for ${VendorName}".failNel
+      case (Some(body), _) => {
```

Comment:
I don't understand why sometime `=>` are aligned and sometimes members of the patmat are aligned, can we prioritize one over the others?